pkg/api/applications.go

```
@@ -370,19 +370,19 @@ func (s *ApplicationService) WatchApplicationsWithOptions(ctx context.Context, e
 		return fmt.Errorf("failed to start watch stream: %w", err)
 	}
 	cblog.With("component", "api").Info("WatchApplications: stream established successfully")
-	
+
 	// Create AccumulatingSSEReader for reliable event processing
 	sseConfig := DefaultSSEConfig()
 	sseReader := NewAccumulatingSSEReader(streamResp.Body, sseConfig)
 	defer sseReader.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `sseReader.Close` is not checked

(errcheck)